sequel-jdbc-as400.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.bindir = 'bin'
 
   s.add_runtime_dependency 'jdbc-jt400', '~> 9.1'
-  s.add_runtime_dependency 'sequel', '~> 5.0'
+  s.add_runtime_dependency 'sequel', '~> 5.11'
```

Comment:
I am hesitant to include this change; it _forces_ all consumers of the gem to upgrade their Sequel version. An individual gem user could still use Sequel 5.11 with the gem; our previous constraint just means that 5.0 is the _minimum_ gem version usable.

So `bundle update sequel` in the consuming project seems to me like a better way than _forcing_ all consumers to upgrade their Sequel version.

If we however go with incorporating this change, it mandates a major version bump IMO, since the change is invasive to our consumers.

Reply:
(misclicked a new button!)

Reply:
That's right!

This blog article explains:
https://robots.thoughtbot.com/rubys-pessimistic-operator

> `~> 1.1` means that when you bundle install, you’ll get the highest-released gem version of thin between the range `>= 1.1` and `< 2.0`.

So, we need no new release.

Great!